scss/vars.scss

```diff
@@ -0,0 +1,3 @@
+$colorTextPrimary: white;
```

Comment:
it is better to write css variables in kebab-case

Reply:
done